docs/vmanomaly.md

```diff
@@ -206,6 +206,7 @@ vm_license_expires_in_seconds 4.886608e+06
 ```
 
 Example alerts for [vmalert](https://docs.victoriametrics.com/vmalert.html):
+{% raw %}
```

Comment:
the new doc doesn't support this, could you pleas tell me the reason behind and can we fix this via CSS?

Reply:
This is how it looks like atm
![1696598615](https://github.com/VictoriaMetrics/VictoriaMetrics/assets/1367798/e035da9b-383c-41d1-91bf-f81704c2b677)

Reply:
ok, in that case we need to add automation to remove this from new docs